Utils/build/lib/dataset/dataset.py

```diff
@@ -0,0 +1,74 @@
+from collections import namedtuple
+from typing import List
+
+annotation = namedtuple('annotation', 'tag, start, end, ne')
+
+
+class Dataset:
+    def __init__(self):
+        self.dataset = []  # храним набор tuple (text, [(annotation, start, end, ne)])
```

Comment:
Move this comment into function description: 

`def foo():
"""
Do smth
"""`